### PR TITLE
added spaces between the badges

### LINKED ISSUE #5520: Add spacing on Recognition page
### Current Behavior
There is no spacing between the badges  on the [recognition page](https://layer5.io/community/handbook/recognition)
image

### Desired Behavior
Add spacing between the badges of `1rem`

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by bunnyDasari**
**Description**

This PR fixes #5520

added spaces between the badges 

before the changes
Screenshot 2024-07-07 082513

after the changes

Screenshot 2024-07-06 111530
Screenshot 2024-07-06 111543

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d90f33c379ef0e7888c2eed56fc921df316cd686 at: https://668a07974212c367a7728f85--layer5.netlify.app

**Comment by Deepak0320Singhal** [comment]
@bunnyDasari Thank you for your contribution.
Let's discuss this on Websites's call. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit), if you would :)

**Comment by Ashparshp** [comment]
@bunnyDasari Make sure to sign-off commits please..

File: src/sections/Community/Handbook/recognition.js [hunk 1]
```diff
@@ -104,96 +108,96 @@ const RecognitionPage = () => {
             </p>
             <ul style={badgeListStyle}>
               <p><b>Activity badges:</b></p>
-              <li>
+              <li style = {badgeSpace}>
```
Comment (sudhanshutech) [inline src/sections/Community/Handbook/recognition.js:111]: @bunnyDasari isn't it seems very repetitive . There could any other way to style it instead passing repetitive styles each time.

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

File: src/sections/Community/Handbook/recognition.js (hunk 1 above)
Comment (bunnyDasari) [reply src/sections/Community/Handbook/recognition.js:111]: I have tried using styled components I think their maybe other way

**Comment by leecalcote** [comment]
Thanks for contributing, @bunnyDasari! Will you be signing your commit as requested?

@hargunkaur286 @vishalvivekm, do we really want this extra space between the badges and the text? 🤔

**Comment by leecalcote** [comment]
Not hearing from @bunnyDasari by the end of Monday's Websites meeting, please consider this PR abandoned by the author and close, @Ashparshp. Keep your fingers-crossed that we hear from @bunnyDasari, though.

**Comment by l5io** [comment]
🚀 Preview for commit 7480bd985ecf14b143a3d6eb35e61a4ba71986f0 at: https://66a00ec0bc242f360824ec19--layer5.netlify.app

**Comment by sudhanshutech** [comment]
Thanks @bunnyDasari , the issue has been fixed already. Lets get you other issues

**Pull request closed without merging**